doc/glossary.rst

```diff
@@ -297,17 +297,18 @@
                 Device", so it is important to know which meaning is intended. 
 
 	OSD fsid
-                This is a unique identifier used to identify an OSD. It is
-                found in the OSD path in a file called ``osd_fsid``. The
-                term ``fsid`` is used interchangeably with ``uuid``
+                The OSD fsid is a unique identifier that is used to identify an
+                OSD. It is found in the OSD path in a file called ``osd_fsid``.
+                The term ``fsid`` is used interchangeably with ``uuid``.
 
 	OSD id
-                The integer that defines an OSD. It is generated by the
-                monitors during the creation of each OSD.
+                The OSD id is the integer that defines an OSD. Each OSD id is
```

Comment:
ID is an integer unique to each OSD.  Each OSD ID is assigned by the monitors

Reply:
I was hoping that you'd have a freely offered opinion about id vs. ID, and that it would match my own opinion. It does.

Accepted.

Reply:
Sort of an initialism, figure it should be uppercased.

Monsters from the id.